src/senaite/core/exportimport/setupdata/__init__.py

```diff
@@ -430,9 +430,10 @@ def Import(self):
                 username = safe_unicode(row['Username']).encode('utf-8')
                 passw = row['Password']
                 if not passw:
-                    warn = "Lab Contact: No password defined for user '{0}' in row {1}. Password established automatically to '{3}'".format(username, str(rownum), username)
-                    logger.warning(warn)
                     passw = username
```

Comment:
out of scope: should we generate a random password instead for better security?

Reply:
Not necessary at this place, because the Excel import will probably go away anyways. But thanks for the heads-up!

Reply:
>  the Excel import will probably go away anyways

@ramonski good to know—do you have a pointer where I can read about what it will be replaced with?

Reply:
It will be probably replaced with the GenericSetup content exporter, see code example here:
https://github.com/senaite/senaite.core/blob/2.x/src/senaite/core/tests/layers.py#L78

So instead of editing an Excel, all the changes are done inside the instance and the whole configuration can then be exported.

The same is possible currently over the `ZMI -> portal_setup -> Export` with the option `SENAITE: Export Content Structure` selected:

<img width="1775" alt="Add-on and configuration management 2022-08-01 11 PM-14-48" src="https://user-images.githubusercontent.com/713193/182247876-24d68fa9-df52-46c1-9100-431e0343d490.png">

Code is located here: https://github.com/senaite/senaite.core/tree/2.x/src/senaite/core/exportimport/genericsetup

Reply:
thanks—and also that seems a good decision to me :+1: